addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -778,6 +779,42 @@ function normalizeRem($editable, rootFontSize=16) {
     }
 }
 
+/**
+ * Convert images of type svg to png.
+ *
+ * @param {JQuery} $editable
+ */
+async function svgToPng($editable) {
+    for (const svg of $editable.find('img[src$=".svg"]')) {
+        // Make sure the svg is loaded before we convert it.
+        await new Promise(resolve => {
+            svg.onload = () => resolve();
+            if (svg.complete) {
+                resolve();
+            }
+        });
+        const image = document.createElement('img');
+        const canvas = document.createElement('CANVAS');
+        const width = _getWidth(svg);
+        const height = _getHeight(svg);
+
+        canvas.setAttribute('width', width);
+        canvas.setAttribute('height', height);
+        canvas.getContext('2d').drawImage(svg, 0, 0, width, height);
+
+        for (const attribute of svg.attributes) {
+            image.setAttribute(attribute.name, attribute.value);
+        }
+
+        image.setAttribute('src', canvas.toDataURL('png'));
+        image.setAttribute('width', width);
+        image.setAttribute('height', height);
+
+        svg.before(image);
```

Comment:
Isn't `before` a jQuery method ? If yes then `svg` should be `$svg`.

Reply:
I was too lazy. I typed "before mdn !" and since it opened the page for the `:before` css selector instead, I thought there were no better match.